com.vrcfury.vrcfury/Editor/VF/Feature/ToggleBuilder.cs

```diff
@@ -69,14 +69,24 @@ public class ToggleBuilder : FeatureBuilder<Toggle> {
         return (model.name, model.usePrefixOnParam);
     }
 
+    private bool getLocalOnly() {

```

Comment:
This method name is misleading. It's called getLocalOnly, but you're returning the opposite.

Reply:
fixed